pom.xml

```diff
@@ -1057,6 +1057,30 @@ Additionally it includes CPD, the copy-paste-detector. CPD finds duplicated code
             </build>
         </profile>
 
+        <profile>
+            <id>javadoc-aggregate</id>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-javadoc-plugin</artifactId>
+                            <configuration>
+                                <additionalDependencies>
+                                    <additionalDependency>
+                                        <groupId>net.sourceforge.pmd</groupId>
+                                        <artifactId>pmd-apex-jorje</artifactId>
+                                        <version>${project.version}</version>
+                                        <classifier>lib</classifier>
+                                    </additionalDependency>
+                                </additionalDependencies>
```

Comment:
We should also include

```xml
<excludePackageNames>*.internal</excludePackageNames>
````

As those are not intended to be consumed / known by users / developers